### ISSUE TITLE
Supabase auth always overwrites the window location during restoreAuth

### ISSUE THREAD

**Issue by KrisCoulson** [redwoodjs/redwood#9549]
This line below always overwrites the window location any time a redwood app loads. It is meant to handle refreshing the auth tokens.
https://github.com/redwoodjs/redwood/blob/43227f3b66a66c304ac2dbe513024a3ebc1ae145/packages/auth-providers/supabase/web/src/supabase.ts#L208-L212

It always gets called because this useEffect calls restoreAuth
https://github.com/redwoodjs/redwood/blob/43227f3b66a66c304ac2dbe513024a3ebc1ae145/packages/auth/src/AuthProvider/AuthProvider.tsx#L130C1-L138

This issue is this completely wipes out search params on page load even if they are unrelated to supabase authentication.

**Comment by Tobbe** [comment]
Thanks @KrisCoulson. Do you want to take a look at fixing it? No pressure, just let me know 🙂

**Comment by KrisCoulson** [comment]
Yeah @Tobbe I can take a look at this. I self assigned. Not 100% sure when I can get to it ATM but I just wanted to get it documented here.

**Comment by Tobbe** [comment]
Yeah, no rush, and thank you! 🙏

**Comment by Tobbe** [comment]
Finally back home again at 2 am this morning :D I'll try to get a release out with this fix the coming week. It'll probably be the last RedwoodJS release. Further updates and releases will happen over at https://cedarjs.com

**Comment by anton-clad** [comment]
If this is the way, we'll start transitioning @Tobbe. Thank you
